src/types/library/library_item.rs

```diff
@@ -112,6 +120,10 @@ pub struct LibraryItemState {
     #[serde(default, rename = "video_id")]
     #[serde_as(deserialize_as = "DefaultOnNull<NoneAsEmptyString>")]
     pub video_id: Option<String>,
+    /// The last stream watched.
+    #[serde(default, rename = "stream_id")]
+    #[serde_as(deserialize_as = "DefaultOnNull<NoneAsEmptyString>")]
+    pub stream_id: Option<String>,
```

Comment:
Let's keep the camelCase. This is very weird, by default we camelCase the fields, but on this and the `video_id` we use snake_case?!
This should be fixed, we need consistency.

Reply:
I kept it because the other fields are lowercase, it's related to the way the api works